### PR TITLE
Revert "Revert "Assignments : assignment 1 completed""

### PR DESCRIPTION
**Pull request by Jal247**
Revert of revert Jal247/shell#3

### CODE REVIEW

File: 02_activities/assignments/assignment.sh [hunk 1]
```diff
@@ -8,17 +8,30 @@ set -x
...
 # 5. Verify that file3 contains the words "hello world" by printing the contents of the file in the terminal
+$cat dir2/file3.txt
 
 # 6. Delete file4
+$rm dir2/file4.txt
+$rm -i dir2/file4.txt   # with confirmation 
 
 # 7. Delete directories dir4 and dir5 including all their contents (if any)
+$rm -r dir4/ dir5/
+$rm -r -i -v dir4/ dir5/   # with confirmation and updation
```
Comment (dwz92) [inline 02_activities/assignments/assignment.sh]: remove the $ before your commands

File: 02_activities/assignments/assignment.sh [hunk 2]
```diff
@@ -8,17 +8,30 @@ set -x
...
 
 # 8. List the contents of the parent directory to verify the deletion of dir4 and dir5
+$ls
+   OR
+$cd ..
+$ls assignment1/
```
Comment (dwz92) [inline 02_activities/assignments/assignment.sh]: remove the $ and choose one answer

File: 02_activities/assignments/assignment.sh (hunk 1 above)
Comment (Jal247) [reply 02_activities/assignments/assignment.sh]: Hello! Thank you for the comment.
 In the branch 'rever-3-revert-2-assignments', I have removed $ from the commands in my assignment.sh file. Also commit changes. Is this good or Do I need to create a new pull request?

**Review by dwz92 — APPROVED** [review]
Good

**Review by simeon-demo — APPROVED** [review]

**Pull request merged**